README.md

```diff
@@ -143,18 +145,24 @@ nextflow run phac-nml/monkeypox-nf --help
         - Column headers were renamed such that `.` were turned into `_` and all are prefixed with `nc_`
         - `,` separators inside the cells were changed to `;`
 
-    | sample | num_reads_mapped | avg_read_depth | total_n_count | proportion_basecalled | nc_clade | ... |
+    | sample | num_reads_mapped | mean_sequencing_depth | median_sequencing_depth | num_consensus_n | genome_completeness | nc_clade | ... |
     |-|-|-|-|-|-|-|
-    | Sample1 | 7606 | 4.47005 | 194674 | 0.0128544 | hMPXV-1 | ... |
-    | Sample2 | 320700 | 213.525 | 423 | 0.997855 | hMPXV-1 | ... |
+    | Sample1 | 7606 | 4.47005 | 6 | 194674 | 0.0128544 | hMPXV-1 | ... |
+    | Sample2 | 320700 | 213.525 | 192 | 423 | 0.997855 | hMPXV-1 | ... |
 
     - Non-nextclade columns are as follows:
     ```
-    sample                - Name of the sample
-    num_reads_mapped      - Int number of reads mapped, pulled from BAM file using samtools flagstats. Remember reads are paired
-    avg_read_depth        - Float average genomic read depth, pulled from BAM file using samtools depth and awk
-    total_n_count         - Int number of positions that were not basecalled in the genome, from seqtk comp
-    proportion_basecalled - Float proportion of the genome called a base (genome completeness), from seqtk comp and awk
+    sample                  - [String] name of the sample
+    num_reads_mapped        - [Int] number of total reads mapped, pulled from BAM file using samtools flagstats.
+                                Reads are paired so the number of total sequence fragments can be obtained by dividing this value by 2
+    mean_sequencing_depth   - [Float] average genomic read depth, pulled from BAM file using samtools depth and awk
```

Comment:
Change **average** to **mean**

Reply:
Also, it's been a while since I ran `samtools depth`, but do you know if it reports all positions (including those with a depth of 0). Or does it exclude those with a depth of 0. Those could have an impact in your mean/median calculations and I have a vague recollection of being concerned about this when using `samtools depth` long ago but I forget what the default was.

Reply:
Will make that change from average -> mean! Thanks

For samtools depth, using the `-a` flag it does output all positions so the calculations should be correct
```bash
AVG_PLUS_MEDIAN_COV=`samtools depth -a $filteredbam | \
        sort -n -k 3 | \
        awk '{ OFS = "," } {dep_a[i++]=\$3;count++;sum+=\$3} END {if (count=='0') {print "0,0"} else print sum/count,dep_a[int(i/2)]}'`
```

Reply:
Awesome. sounds good. Thanks for clarifying about the `-a` flag 👍